GAE/src/org/waterforpeople/mapping/dataexport/RawDataSpreadsheetImporter.java

```diff
@@ -240,8 +241,13 @@ public void executeImport(File file, String serverBase, Map<String, String> crit
 
         //these are all for the base sheet
         Map<Integer, Long> columnIndexToQuestionId = sheetMap.get(baseSheet);
-        int firstQuestionColumnIndex = Collections.min(columnIndexToQuestionId.keySet());
-        Map<String, Integer> metadataColumnHeaderIndex = getMetadataColumnIndex(baseSheet, firstQuestionColumnIndex, headerRowIndex, false);
+        int firstQuestionColumnIndex = 0;
```

Comment:
I propose we initialize it to `-1` to indicate that no question column has been found rather than defaulting to the first column. 

```suggestion
        int firstQuestionColumnIndex = -1;
```